python/cudf/dataframe/multiindex.py

```diff
@@ -218,6 +271,8 @@ def is_contiguous(self):
 
     @property
     def size(self):
+        if self._codes is None:
+            self._compute_levels_and_codes()
```

Comment:
You shouldn't need this here. The call of `self.codes[0]` should automatically trigger the computation.

Reply:
Yeah totally

Reply:
Should be removed everywhere now.